kano_profile/profile.py

```diff
@@ -203,14 +203,14 @@ def set_avatar(subcat, item, sync=False):
 
 
 ENVIRONMENT_REMAP = {
-    "Arcade Hall": "arcade_hall",
-    "Concert Hall": "concert_hall",
-    "Jungle": "jungle",
-    "Space": "space",
-    "Dojo": "dojo",
-    "Park": "park",
-    "Video Set": "video_set",
-    "Minecraft": "minecraft"
+    _("Arcade Hall"): 'arcade_hall',
+    _("Concert Hall"): 'concert_hall',
+    _("Jungle"): 'jungle',
+    _("Space"): 'space',
+    _("Dojo"): 'dojo',
+    _("Park"): 'park',
+    _("Video Set"): 'video_set',
+    _("Minecraft"): 'minecraft'
```

Comment:
The keys here are being translated but the [`get_environment()`](https://github.com/KanoComputing/kano-profile/blob/650d21589eb91ba44a18114511393e0e964c3136/kano_profile/profile.py#L222-L230) function is retrieving the environments from the saved profile (which will and should be in English) and comparing them with the `ENVIRONMENT_REMAP` keys which is likely to fail.